### PR TITLE
charts for 3.8.3

### PR DESCRIPTION
**Pull request by elnyry**
admin handler, related config

### CODE REVIEW

File: centralledger/chart-handler-admin-transfer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,704 @@
...
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+containers:
+  api:
+    image:
+      repository: mojaloop/central-ledger
+      tag: v3.8.1-snapshot
+      pullPolicy: Always
+      command: '["node", "src/handlers/index.js", "handler", "--prepare"]'
```
Comment (mdebarros) [inline centralledger/chart-handler-admin-transfer/values.yaml]: This should be updated. Should this not be --admin?

File: centralledger/chart-handler-timeout/values.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-handler-timeout/values.yaml]: Shouldn't this be v3.8.1?

File: centralledger/chart-handler-transfer-fulfil/values.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-fulfil/values.yaml]: Snapshot?

File: centralledger/chart-handler-transfer-get/values.yaml [hunk 4]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-get/values.yaml]: Snapshot?

File: centralledger/chart-handler-transfer-position/values.yaml [hunk 5]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-position/values.yaml]: Snapshot?

File: centralledger/chart-handler-transfer-prepare/values.yaml [hunk 6]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-handler-transfer-prepare/values.yaml]: Snapshot?

File: centralledger/chart-service/values.yaml [hunk 7]
```diff
@@ -24,7 +24,7 @@ containers:
   api:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-service/values.yaml]: Snapshot?

File: centralledger/chart-service/values.yaml [hunk 8]
```diff
@@ -49,7 +49,7 @@ containers:
   admin:
     image:
       repository: mojaloop/central-ledger
-      tag: v3.8.0
+      tag: v3.8.1-snapshot
```
Comment (mdebarros) [inline centralledger/chart-service/values.yaml]: Snapshot?

File: centralledger/chart-handler-admin-transfer/values.yaml (hunk 1 above)
Comment (elnyry) [reply centralledger/chart-handler-admin-transfer/values.yaml]: updated

**Pull request merged**